Components/home.css

```diff
@@ -0,0 +1,298 @@
+body {
+    height: 312vw;
+    max-width: 100%;
+    overflow-x: hidden;
+}
+
+.nav-bar {
+  min-height: 12vh;
```

Comment:
You should add a max-height for your navbar. It's too big on large screens

Reply:
Ok I'll do it